app/cells/plugins/core/text_cell.rb

```diff
@@ -27,6 +27,10 @@ def input_styles
         input_display&.[](:styles)
       end
       
+      def number_step
```

Comment:
`TextCell` seems to still include references to numericality - these should be removed